### PR TITLE
Coding standards fix

### PR DESCRIPTION
**Pull request by giorello**
Ref. #779

### CODE REVIEW

File: app/code/community/Ebizmarts/MailChimp/Model/Email/Queue.php [hunk 1]
```diff
@@ -101,60 +60,115 @@ public function send()
...
-                    }
+        return $this;
+    }
 
-                    $mailer->setSubject('=?utf-8?B?' . base64_encode($parameters->getSubject()) . '?=');
-                    $mailer->setFrom($parameters->getFromEmail(), $parameters->getFromName());
-                    if ($parameters->getReplyTo() !== null) {
-                        $mailer->setReplyTo($parameters->getReplyTo());
-                    }
+    protected function _sendMandrillEnabled($message, $parameters, $storeId)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Email/Queue.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Email/Queue.php [hunk 2]
```diff
@@ -101,60 +60,115 @@ public function send()
...
+            );
+            $mailer->send();
+        } catch (Exception $e) {
+            Mage::logException($e);
+        }
+
+        unset($mailer);
+    }
+
+    protected function _sendMandrillNotEnabled($message, $parameters)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Email/Queue.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Email/Template.php [hunk 3]
```diff
@@ -66,111 +60,143 @@ public function send($email, $name = null, array $variables = array())
...
                 'type' => 'bcc'
             );
         }
 
-        $email['from_name'] = $this->getSenderName();
-        $email['from_email'] = $this->getSenderEmail();
+        return $to;
+    }
+
+    protected function _getEmailFrom($mail)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Email/Template.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Mysql4/Interestgroup.php [hunk 4]
```diff
@@ -23,13 +23,18 @@ public function _construct()
         $this->_init('mailchimp/interestgroup', 'id');
     }
 
-    public function getByRelatedIdStoreId($customerId = 0, $subscriberId = 0, $storeId)
+    public function getByRelatedIdStoreId($customerId = 0, $subscriberId = 0, $storeId = 0)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Mysql4/Interestgroup.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php [hunk 5]
```diff
@@ -244,30 +256,44 @@ public function _profile(array $data)
...
-                    } catch (Exception $e) {
-                        $helper->logError($e->getMessage());
-                    }
+                    $this->_addSubscriberData($subscriber, $fname, $lname, $email, $listId);
                 }
             }
         }
     }
 
+    protected function _addSubscriberData($subscriber, $fname, $lname, $email, $listId)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php [hunk 6]
```diff
@@ -12,7 +12,8 @@ class Ebizmarts_MailChimp_Model_Subscriber extends Mage_Newsletter_Model_Subscri
 
     public function sendUnsubscriptionEmail()
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php [hunk 7]
```diff
@@ -21,7 +22,8 @@ public function sendUnsubscriptionEmail()
 
     public function sendConfirmationRequestEmail()
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php [hunk 8]
```diff
@@ -30,7 +32,8 @@ public function sendConfirmationRequestEmail()
 
     public function sendConfirmationSuccessEmail()
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/Subscriber.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Model/System/Config/Backend/List.php [hunk 9]
```diff
@@ -20,19 +20,34 @@ protected function _afterSave()
...
+            );
             $this->getAdminSession()->addWarning($message);
         }
 
         if ($valueChanged && ($moduleIsActive || $thisScopeHasSubMinSyncDateFlag) && $this->getValue()) {
-            $configValues = array(array(Ebizmarts_MailChimp_Model_Config::GENERAL_SUBMINSYNCDATEFLAG, Varien_Date::now()));
+            $configValues = array(
+                array(
+                    Ebizmarts_MailChimp_Model_Config::GENERAL_SUBMINSYNCDATEFLAG,
+                    Mage::getSingleton('core/date')->gmtDate("Y-m-d H:i:s")
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/System/Config/Backend/List.php]: Is date okay?

File: app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Account.php [hunk 10]
```diff
@@ -213,7 +238,11 @@ protected function makeHelper()
 
     protected function getDateSync($mailchimpStoreId)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Account.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/controllers/Adminhtml/MailchimpController.php [hunk 11]
```diff
@@ -237,7 +255,10 @@ protected function getSourceListOptions($apiKey, $mcStoreId)
...
     protected function getSourceInterestOptions($apiKey, $listId)
     {
-        return Mage::getModel('Ebizmarts_MailChimp_Model_System_Config_Source_CustomerGroup', array('api_key' => $apiKey, 'list_id' => $listId))->toOptionArray();
+        return Mage::getModel(
+            'Ebizmarts_MailChimp_Model_System_Config_Source_CustomerGroup',
+            array('api_key' => $apiKey, 'list_id' => $listId)
+        )->toOptionArray();
     }
 
     protected function getApiKeyValue()
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/controllers/Adminhtml/MailchimpController.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/controllers/WebhookController.php [hunk 12]
```diff
@@ -88,4 +72,29 @@ public function indexAction()
             }
         }
     }
+
+    protected function _deleteWebhook($storeId, $listId)
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/controllers/WebhookController.php]: Add comment for params and return.

File: app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/Sales/Order/View/Info/Monkey.php [hunk 13]
```diff
@@ -17,12 +17,12 @@ class Ebizmarts_MailChimp_Block_Adminhtml_Sales_Order_View_Info_Monkey extends M
...
-    protected $campaignName = null;
+    protected $_campaignName = null;
 
     /**
      * @var Mage_Sales_Model_Order $order
      */
-    protected $order = null;
+    protected $_order = null;
 
     public function isReferred()
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/Sales/Order/View/Info/Monkey.php]: Solved

File: app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/Sales/Order/Grid/Renderer/Mailchimp.php [hunk 14]
```diff
@@ -10,19 +10,24 @@
  * @date:     7/7/16 1:31 PM
  * @file:     Abandoned.php
  */
-class Ebizmarts_MailChimp_Block_Adminhtml_Sales_Order_Grid_Renderer_Mailchimp extends Mage_Adminhtml_Block_Widget_Grid_Column_Renderer_Abstract
+class Ebizmarts_MailChimp_Block_Adminhtml_Sales_Order_Grid_Renderer_Mailchimp
+    extends Mage_Adminhtml_Block_Widget_Grid_Column_Renderer_Abstract
 {
     public function render(Varien_Object $row)
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/Sales/Order/Grid/Renderer/Mailchimp.php]: Solved.

File: app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/System/Config/Form/Field/Mapfields.php [hunk 15]
```diff
@@ -65,7 +66,7 @@ public function __construct()
     protected function _renderCellTemplate($columnName)
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Block/Adminhtml/System/Config/Form/Field/Mapfields.php]: Solved.

File: app/code/community/Ebizmarts/MailChimp/Block/Checkout/Success/Groups.php [hunk 16]
```diff
@@ -49,14 +49,14 @@ public function getInterest()
 
     public function getMessageBefore()
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Block/Checkout/Success/Groups.php]: Solved.

File: app/code/community/Ebizmarts/MailChimp/Block/Checkout/Success/Groups.php [hunk 17]
```diff
@@ -49,14 +49,14 @@ public function getInterest()
...
     {
-        $storeId = $this->storeId;
+        $storeId = $this->_storeId;
         $message = $this->getMailChimpHelper()->getCheckoutSuccessHtmlBefore($storeId);
         return $message;
     }
 
     public function getMessageAfter()
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Block/Checkout/Success/Groups.php]: Solved.

File: app/code/community/Ebizmarts/MailChimp/Helper/Data.php [hunk 18]
```diff
@@ -4155,7 +4670,7 @@ protected function getCustomerSession()
      */
     protected function getCurrentDateTime()
     {
-        return Mage::getModel('core/date')->date('d-m-Y H:i:s');
+        return Mage::getModel('core/date')->gmtDate('d-m-Y H:i:s');
```
[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: The method date($format, $input) calls to 'timestamp($input = null)' and when $input is null, this method returns the result of calling to 'gmtTimestamp()'; the Method gmtDate, calls directly to gmtTimestamp(). That makes this both methods return the same.

[the remark this answers is not in the document]
Comment (giorello) [reply app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: Anyways, this should be using the Helper method 'formatDate'.

**Pull request merged**